app/views/devise/passwords/new.html.haml

```diff
@@ -4,7 +4,7 @@
   = bootstrap_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
     = devise_error_messages!
 
-    = f.text_field :login, autofocus: true, label: "User name"
+    = f.text_field :email, autofocus: true, label: "Email address"
```

Comment:
Hmm, wonder if this renders as an email field (with browser-side validation)

Reply:
Now it does.